src/parcsr_ls/par_nongalerkin.c

```diff
@@ -1689,14 +1686,17 @@ hypre_BoomerAMGBuildNonGalerkinCoarseOperator( hypre_ParCSRMatrix **RAP_ptr,
 
    /* Initialize the ijmatrix, leveraging our knowledge of the nonzero
     * structure in Pattern */
-   ierr += HYPRE_IJMatrixCreate(comm, first_col_diag_RAP, last_col_diag_RAP,
-                                first_col_diag_RAP, last_col_diag_RAP, &ijmatrix);
-   ierr += HYPRE_IJMatrixSetObjectType(ijmatrix, HYPRE_PARCSR);
+   HYPRE_IJMatrixCreate(comm, first_col_diag_RAP, last_col_diag_RAP,
```

Comment:
@victorapm why is the error checking removed from here and other places?

Reply:
If we want to access the current error code in hypre, we should use `HYPRE_GetError`. Therefore, we don't need to save error codes returned from HYPRE functions as done here